styles/globals.css

```
@@ -781,3 +781,63 @@ article.report-article tbody tr:hover td {
+  /* Browsers strip background colors/images by default — opt back in
+   * so card backgrounds, chart fills, and accent colors print. */
+  * {
+    -webkit-print-color-adjust: exact !important;
+    print-color-adjust: exact !important;
+  }
+  /* Don't split a chart card across pages — keeps each metric block
+   * intact even if it pushes onto the next sheet. */
+  .chart-card,
+  .report-print-no-break {
+    break-inside: avoid;
+    page-break-inside: avoid;
```

Stylelint (17.12.0)

[error] Expected "page-break-inside" to be "break-inside" (property-no-deprecated)

(property-no-deprecated)